cekit/generator/base.py

```diff
@@ -53,10 +53,14 @@ def __init__(self, descriptor_path, target, overrides):
 
         if overrides:
             for override in overrides:
-                # TODO: If the overrides is provided as text, why do we try to get path to it?
                 LOGGER.debug("Loading override '{}'".format(override))
+
+                override_artifact_dir = os.path.dirname(os.path.abspath(override))
+                if not os.path.exists(override):
+                    # Just use current directory for the artifact_dir
+                    override_artifact_dir = os.getcwd()
```

Comment:
You are changing the logic here which will break runs in case the overrides in not in current directory. Saying it differently - artifact dir should be relative to the overrides file which is defining it. You are using current directory no matter where the override file is located.

Other thing is that current directory for inline overrides is probably not a good choice either. It should be directory where the image definition is located.